packages/web-api/src/controllers/api-controller.spec.ts

```diff
@@ -0,0 +1,220 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import 'reflect-metadata';
+
+import { Context } from '@azure/functions';
+import { ApiController } from './api-controller';
+
+export class ApiControllerMock extends ApiController {
+    public readonly apiVersion = '1.0';
+    public invoked = false;
+
+    protected invokeImpl(): void {
+        this.invoked = true;
+    }
+}
+
+describe('validateContentType()', () => {
+    it('should validate HTTP POST and PUT only', () => {
```

Comment:
> it('should validate HTTP POST and PUT only', () => { [](start = 3, length = 53)

you should also have a test for PUT request

Reply:
updated